test/winding.jl

```diff
@@ -11,6 +11,17 @@
   @test winding(p, reverse(c)) ≈ T(-2)
   @test winding([p, p], c) ≈ T[2, 2]
 
+  p = Point(LatLon(T(0.5), T(0.5)))
+  c = Ring(Point.([LatLon(T(0), T(0)), LatLon(T(1), T(0)), LatLon(T(1), T(1)), LatLon(T(0), T(1))]))
+  @test winding(p, c) ≈ T(1)
+  @test winding(p, reverse(c)) ≈ T(-1)
+  @test winding([p, p], c) ≈ T[1, 1]
+
+  # error: both arguments must have the same CRS
+  p = Point(LatLon(T(0.5), T(0.5)))
+  c = Ring(point.([(0, 0), (1, 0), (1, 1), (0, 1), (0, 0), (1, 0), (1, 1), (0, 1)]))
```

Comment:
similarly, `point` is an alias to `Point` with `Cartesian` coords in the tests. Maybe we can explore the idea of introducing a `cart` helper with the appropriate machine type, and a `pointcart(x, y) = Point(cart(x, y))` to make it more explicit for readers.